### PR TITLE
[MIG] website_img_bg_style: Handle module removal

### PR DESCRIPTION
**Pull request by yajo**
This module is removed because the features it added are now upstream.

Applying needed conversion to keep consistent styling and UX.

Replaces #1814.

WIP until merged:

- [x] https://github.com/OCA/openupgradelib/pull/148

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
@Tecnativa

### CODE REVIEW

File: addons/website/migrations/10.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -1,7 +1,79 @@
 # -*- coding: utf-8 -*-
 # © 2017 Therp BV <http://therp.nl>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from itertools import product
+
 from openupgradelib import openupgrade
+from openupgradelib.openupgrade_tools import (
```
Comment (pedrobaeza) [inline addons/website/migrations/10.0.1.0/pre-migration.py:7]: Is this compatible with Python 2?

Comment (yajo) [reply addons/website/migrations/10.0.1.0/pre-migration.py:7]: Yes, it's `openupgrade_120` that isn't.

**Comment by pedrobaeza** [comment]
It seems there's a incompatibility between versions in `openupgradelib` or here:
```
AttributeError: 'module' object has no attribute 'tools'
```

**Comment by MiquelRForgeFlow** [comment]
it seems to me that is openupgradelib or travis files :S

**Comment by yajo** [comment]
Restarted Travis build, now that https://github.com/OCA/openupgradelib/pull/148 is merged.

**Comment by pedrobaeza** [comment]
@Yajo I already did that. Please check the previous log with the problem.

**Comment by yajo** [comment]
I can see no relation between my patch and those errors, apart from the fact that it's pretty outdated. I rebased; let's see.

**Comment by pedrobaeza** [comment]
I would say that it has to be with the import you are doing, that provokes another imports in the target file, and the order/path mangling in Odoo doesn't fit exactly.

File: addons/website/migrations/10.0.1.0/pre-migration.py [hunk 2]
```diff
@@ -1,7 +1,79 @@
...
+    REPLACEMENTS.append(_r(
+        "bg-size-%s-h bg-size-%s-v" % (h.strip("%"), v.strip("%")),
+        "oe_img_bg o_bg_img_opt_custom",
+        styles={"background-size": "%s %s" % (h, v)}))
+
+
+def update_website_views(env):
+    """Handle removal of website_img_bg_style module."""
+    # Find report views, which should never be converted here
+    report_names = env["ir.actions.report"].search([
```
Comment (MiquelRForgeFlow) [inline addons/website/migrations/10.0.1.0/pre-migration.py]: -> `ir.actions.report.xml`

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
All attended.

**Review by MiquelRForgeFlow — APPROVED** [review]

File: odoo/addons/openupgrade_records/lib/apriori.py [hunk 3]
```diff
@@ -35,6 +35,8 @@
     # OCA/stock-logistics-workflow:
     'product_customer_code_picking':
         'product_supplierinfo_for_customer_picking',
+    # OCA/website
+    'website_img_bg_style': 'website',
```
Comment (MiquelRForgeFlow) [inline odoo/addons/openupgrade_records/lib/apriori.py]: wrong, this should go below, in the `merged_modules` section

**Review by MiquelRForgeFlow — DISMISSED** [review]

File: odoo/addons/openupgrade_records/lib/apriori.py (hunk 3 above)
Comment (pedrobaeza) [reply odoo/addons/openupgrade_records/lib/apriori.py]: Please fix this

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: odoo/addons/openupgrade_records/lib/apriori.py (hunk 3 above)
Comment (pedrobaeza) [reply odoo/addons/openupgrade_records/lib/apriori.py]: I have changed this myself.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**